crates/cli/src/opts/build/core.rs

```diff
@@ -120,11 +120,7 @@ pub struct BuildOpts {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub build_info_path: Option<PathBuf>,
 
-    /// Use EOF-enabled solc binary. Enables via-ir and sets EVM version to Prague. Requires Docker
-    /// to be installed.
-    ///
-    /// Note that this is a temporary solution until the EOF support is merged into the main solc
-    /// release.
+    /// Build with EOF.
```

Comment:
Should mention that this compiles contracts to EOF bytecode

Reply:
added in https://github.com/foundry-rs/foundry/pull/10081/commits/165f64e6e7ca563d40964a016bade30f46ede790